classes/profile.php

```diff
@@ -212,6 +212,10 @@ public function save_record(): int {
             $this->raw_set('userid', $USER->id);
         }
 
+        if ($this->check_update_courseid($COURSE->id)) {
```

Comment:
Is this going to store everything in the admin level under course=1? That might be fine, or it might muddy the waters a little

Reply:
Yes correct. I would expect anyone using the Excimer plugin has enough Moodle knowledge to know why course id = 1 is used so often so I don't think it's an issue.